clients/src/main/java/org/apache/kafka/clients/ClusterConnectionStates.java

```diff
@@ -34,19 +36,22 @@
  *
  */
 final class ClusterConnectionStates {
-    private final long reconnectBackoffInitMs;
-    private final long reconnectBackoffMaxMs;
-    private final static int RECONNECT_BACKOFF_EXP_BASE = 2;
```

Comment:
why did we remove the constance and hard-code the number inline?

Reply:
I was thinking that the existing code hardcoded the jitter. I added back the constance and also added a constance for jitter.